packages/transfer-app/src/SendBalance/validate.ts

```diff
@@ -5,14 +5,17 @@
 import ApiRx from '@polkadot/api/rx';
 import { SubmittableExtrinsic } from '@polkadot/api/submittable/types';
 import { getTypeRegistry } from '@polkadot/types';
-import { MAX_SIZE_BYTES, MAX_SIZE_MB } from '@polkadot/react-signer/Checks/constants';
 import { compactToU8a } from '@polkadot/util';
 import BN from 'bn.js';
 import { Either, left, right } from 'fp-ts/lib/Either';
 import { none, some } from 'fp-ts/lib/Option';
 
 import { AllExtrinsicData, Errors, SubResults, UserInputs, WithAmount, WithAmountExtrinsic } from './types';
 
+// FIXME This code is identical to `packages/ui-common/src/util`!
```

Comment:
I've moved this stuff in https://github.com/paritytech/substrate-light-ui/pull/581/files